src/game_api/flags.hpp

```diff
@@ -85,8 +85,8 @@ const char* more_flags[]{
     "9: ",
     "10: ",
     "11: Swimming",
-    "12: ",
-    "13: ",
+    "12: Hit ground (bomb)",
+    "13: Hit wall (bomb)",
```

Comment:
This is probably for all thowable entities, not only bomb. Never got time to test it fully